docs/root/configuration/http/http_filters/credential_injector_filter.rst

```diff
@@ -71,3 +86,18 @@ The HTTP credential injector filter outputs statistics in the ``http.<stat_prefi
   ``injected``, Counter, Total number of requests with injected credentials
   ``failed``, Counter, Total number of requests that failed to inject credentials
   ``already_exists``, Counter, Total number of requests that already had credentials and overwrite is false
+
+OAuth2 Client Credential Injector extension specific statistics are also emitted in the ``http.<stat_prefix>.credential_injector.oauth2.`` namespace.
```

Comment:
```suggestion
OAuth2 client credential injector extension specific statistics are also emitted in the ``http.<stat_prefix>.credential_injector.oauth2.`` namespace.
```

the name of the filter is a borderline case - "proper" names should be capitalized - but i think its more consistent (and preferable) to use normal caps